### ISSUE TITLE
Timestamps & useServerTime option

### ISSUE THREAD

**Issue by Squamoto** [jaggedsoft/php-binance-api#40]
I am getting a lot of -1021 "Timestamp for this request was 1000ms ahead of the server's time." errors when performing a signed request. According to the API, it is possible to send a useServerTime parameter (it is mentioned in the PHP API as well: "balanceData error: Please make sure your system time is synchronized, or pass the useServerTime option."). With the PHP API it doesn't look like it is possible to use the useServerTime option tho?

**Comment by jaggedsoft** [comment]
Coming very soon. Not finished yet

**Comment by jaggedsoft** [comment]
Available in `v0.0.12` https://github.com/jaggedsoft/php-binance-api/commit/991578f690a9c91a833e0794779dd3e6aeb78c41
#### useServerTime Method #1
```php
$api = new Binance\API("<key>","<secret>", ['useServerTime'=>true]);
```

#### useServerTime Method #2 
```php
// Call this before running other functions
$api->useServerTime();
```
```
